spec/scope_spec.cr

```diff
@@ -0,0 +1,60 @@
+require "./helper"
+
+module PlaceOS::Api
+  base = Zones::NAMESPACE[0]
+
+  with_server do
+    WebMock.reset
+    WebMock.allow_net_connect = true
+
+    describe "default public scope" do
+      test_crd(klass: Model::Zone, controller_klass: Zones)
+    end
+
+    it ".read scope" do
```

Comment:
```crystal
  describe "scope" do
    context "write" do
      ...
    end
    context "read" do
      it "allows access to show" do
      ...
      end
      it "allows access to index" do
      ...
      end
    end
```

Ideally this would be automated via a method or macro, testing the negative and positive case